deploy/deploy_contracts.ts

```diff
@@ -180,6 +180,14 @@ const func: DeployFunction = async function(hre: HardhatRuntimeEnvironment) {
         args: [Controller.address]
     })
 
+    // AIservice registry
+    await contractDeployer.deployAndRegister({
```

Comment:
Since the AIServiceRegistry is a separate contract that wouldn't be a part of the core protocol contracts I don't think this deploy is necessary here and we also wouldn't be registering the AIServiceRegistry address with the Controller for now.

Reply:
I agree if we don't want to include it yet in the mainnet contracts we can get rid of the controller and Proxy for now 👍🏻.